src/components/SharedComponents/Button/button.styles.ts

```diff
@@ -250,20 +250,35 @@ const buttonStyles = css<ButtonStylesProps>`
       `}
     `}
 
-  ${(props) =>
+    ${(props) =>
     props.variant === "buttonStyledAsLink" &&
     css`
       &:focus {
         & ${ButtonStyledAsLinkFocusUnderline} {
           display: block;
         }
       }
-
       :hover:not(:focus) ${ButtonLabel} {
         text-decoration: underline;
       }
     `}
 
+
+    ${(props) =>
+    props.variant === "buttonStyledAsLink" &&
+    props.size === "medium" &&
+    css`
```

Comment:
This block of code makes it quite particular to the banner usecase, we want to keep in. I've make some suggestions in https://github.com/oaknational/Oak-Web-Application/pull/2916